src/multiplication.ts

```diff
@@ -0,0 +1,30 @@
+let inputA = document.querySelector('#numberA') as HTMLInputElement;
+let numberA: number 
+let inputB = document.querySelector('#numberB') as HTMLInputElement;
+let numberB: number
+let button = document.querySelector('.button-multiplication') as HTMLButtonElement;
+let affichageResultat = document.querySelector('#resultat') as HTMLParagraphElement
+let resultat: number
+
+
+function multiplication(a: number, b: number) {
```

Comment:
prefer arrow function
const multiplication= (a: number, b: number) => {
}

Reply:
it's done